docs/attributes-registry/gen-ai.md

```diff
@@ -33,6 +33,10 @@ This document defines the attributes used to describe telemetry in the context o
 
 `gen_ai.system` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
 
-| Value    | Description | Stability                                                        |
-| -------- | ----------- | ---------------------------------------------------------------- |
-| `openai` | OpenAI      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Value          | Description  | Stability                                                        |
+| -------------- | ------------ | ---------------------------------------------------------------- |
+| `openai`       | OpenAI       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `azure_openai` | Azure OpenAI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Do we have to differentiate? Do we have a reliable way to differentiate? 
AFAIK not:

- we don't capture anything specific to azure
- client SDK does not care and the intention is to keep it this way
- domain name is not a reliable way to differentiate